eZ/Bundle/EzPublishCoreBundle/Resources/config/routing/content.yml

```diff
@@ -0,0 +1,3 @@
+_ezplatform_download:
+    path: /content/download/{contentId}/{fieldIdentifier}/{downloadName}
```

Comment:
Actually, the name in the URI would also be used as the name _given_ to the downloaded file. But it's probably not required.